src/main/scala/view/utilities/BunnyImage.scala

```diff
@@ -0,0 +1,120 @@
+package view.utilities
+
+import model.Bunny
+import model.genome.{Alleles, Genes}
+import model.genome.Alleles._
+import scalafx.scene.image.Image
+import view.utilities.ImageType.{ImageType, Jumping, Normal}
+
+object BunnyImage extends Enumeration {
+
+   case class BunnyImage(normalImage: Image,
+                         jumpingImage: Image,
+                         alleles: List[Alleles.AlleleKind]) extends super.Val
+  import scala.language.implicitConversions
+  implicit def valueToBunnyImage(x: Value): BunnyImage = x.asInstanceOf[BunnyImage]
+
+  val BrownHighEarsLongTeethNormalFur: BunnyImage = BunnyImage(
+                                          new Image("/bunnies/brown/high_ears/long_teeth/normal_fur/normal.png"),
+                                          new Image("/bunnies/brown/high_ears/long_teeth/normal_fur/jumping.png"),
+                                          List(BROWN_FUR, HIGH_EARS, LONG_TEETH, SHORT_FUR))
```

Comment:
Qui sarebbe bello usare direttamente un'istanza di Phenotype, anche se mi rendo conto che probabilmente hai usato una lista perchè per il gene JUMP non ha due immagini diverse... si potrebbe tenere sempre il LOW_JUMP, @alessiarocco98 @martaSpadoni voi come lo preferite?

Reply:
A me l'idea di usare Phenotype piace

Reply:
inizialmente farei anche io con l'idea del Phenotype poi in futuro potremo esplicitarlo anche nell'immagine del coniglio e non solo con l'animazione.